notes-with-indexeddb/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+
+    "description": "Examplifies working with IndexedDB in a background page",
+    "manifest_version": 2,
+    "name": "notes-with-indexeddb",
+    "version": "1.0",
+    "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/notes-with-indexeddb",
+    "icons": {
+        "48": "icons/notebook-48.png"
```

Comment:
It would be best to include the icons in SVG format so that they scale better for high resolution displays, and when placed in the "hamburger" menu.